components/molecule/photoUploader/.npmignore

```diff
@@ -0,0 +1 @@
+src
```

Comment:
why should we do this?

Reply:
I'm not sure about this, but It's a common folder to be ignored before publishing the package 📦

Reply:
When publishing the package (component) we only need the compiled version of the component, so there is no need to add the no compiled version (src/*). Otherwise the package will end up bing bigger than what is need.